Makefile

```diff
@@ -18,6 +18,7 @@ DOCKER_RUN_LIBCOMPOSE := docker run --rm -it --privileged $(LIBCOMPOSE_ENVS) $(L
 default: binary
 
 all: build
```

Comment:
@dnephin yes we do, build just build the `build` image (build inception :stuck_out_tongue_closed_eyes:), it doesn't build anything per se (it's `binary` that builds the binary).

*way too many `build` in this sencence*